src/main/assets/locales/avaya-webchat/locale_en-GB.json

```diff
@@ -0,0 +1,64 @@
+{
+  "iconAltText": "Web chat icon",
+  "hideChat": "Hide",
+  "noAvailableAgents": "No Advisors are currently available",
+  "contactCenterClosed": "The Contact Center is closed",
+  "connectionLost": "Connection lost",
+  "connected": "Connected",
+  "failedToSendRequest": "Something failed while sending the request",
+  "form": {
+    "title": "Before I transfer you to an adviser, complete the form below",
+    "nameField": {
+      "title": "Name",
+      "placeholder": "Your name..."
+    },
+    "emailField": {
+      "title": "Email",
+      "placeholder": "Your email...",
+      "validationError": "Please enter a valid email address"
+    },
+    "caseIdField": {
+      "title": "Case Id",
+      "placeholder": "Your case Id..."
+    },
+    "serviceField": {
+      "title": "Service",
+      "placeholder": "Select service..."
+    },
+    "reasonForContactField": {
+      "title": "Reason for contact",
+      "placeholder": "Briefly tell us how we can help you..."
+    },
+    "transcriptField": {
+      "title": "Email the transcript to me"
+    },
+    "startChat": "Start chat"
+  },
+  "chat": {
+    "waitingForParticipant": "Waiting for participant...",
+    "ewtUpdateStatus": "An Advisor will be with you in approximately {{time}}",
+    "ewtUpdateTimestamp": "Last updated: {{update}}",
+    "ewtAgentAvailableSoon": "An Advisor will be with you in less than one minute",
+    "ewtLongWait": "An Advisor will be with you in {{hours}} hours, {{minutes}} minutes and {{seconds}} seconds",
+    "ewtShortWait": "An Advisor will be with you in {{minutes}} minutes and {{seconds}} seconds",
+    "legalAdviceNotice": "Please note, the Web Chat Advisor cannot provide you with legal advice.",
+    "chatEndedNotice": "The customer has ended this conversation.",
+    "messageFieldPlaceholder": "Your message...",
+    "sendMessage": "Send message",
+    "agentIsTyping": "Advisor is typing...",
+    "conversationEnded": "The chat session has ended but you can start a new one if you need more help",
+    "startNewSession": "Start new session",
+    "endChat": "End chat",
+    "prompt": {
+      "message": "If you leave the chat you will lose your place in the queue. Are you sure?",
+      "yes": "Yes",
+      "no": "No"
+    },
+    "offerSurvey": "Send me a feedback survey"
+  },
+  "survey": {
+    "propose": "Would you like to take part in an email survey at the end of your Webchat?",
+    "yes": "Yes",
+    "no": "No"
+  }
+}
```

Comment:
i dont see where this is used or how its used, i assumed the other script.

Reply:
Yeah this is used in the other script. If you're testing it, it's likely that none of the webchat agents are available (cos its using pre prod URL) and the only content you'll see rendered is L4 "No Advisors are currently available"